### ISSUE TITLE
1.4.0.1 badge position problems

### ISSUE THREAD

**Issue by spidfire** [roughike/BottomBar#544]
I use 'com.roughike:bottom-bar:1.4.0.1' because of the functionality.

It goes often wrong when I update the badge when i'm focussing on the tab in question.

If you after you create the tabbar:
![image](https://cloud.githubusercontent.com/assets/156694/19340688/79085fee-912a-11e6-8b04-de75954379ab.png)

When you select it:
![image](https://cloud.githubusercontent.com/assets/156694/19340736/b5df25ba-912a-11e6-9cf1-04b893e94664.png)

After you recreate the tabbar:
![image](https://cloud.githubusercontent.com/assets/156694/19340583/e778792e-9129-11e6-9db1-fc218b12bbad.png)

When you deselect it:
![image](https://cloud.githubusercontent.com/assets/156694/19340563/d027beec-9129-11e6-8811-ad06774a9232.png)

**Comment by leodeleon22** [comment]
I have the same problem with 3 tabs

![image](https://cloud.githubusercontent.com/assets/13181805/19678303/7ab1ca02-9a9d-11e6-870a-9712dd9d1a61.png)

**Comment by spidfire** [comment]
@roughike  Is there any plan to fix this or are you forceing us to use the new version?

**Comment by roughike** [comment]
I'd say use the new version. There's no plans to keep improving the old one.

**Comment by yombunker** [comment]
@roughike @spidfire i would rephrase that and put it like this, there was a plan to fix it and that's why version 2 was created, it does require some effort to update, but, you will get more bang for your bucks. Believe, i spend like 1 day updating our project but, at the end you get better performance and your layouts look better in the editor, cause now you can see what is going to be there.
